cartridges/plugin_beam/cartridge/templates/default/beam/beam_select_nonprofit.isml

```diff
@@ -33,6 +34,57 @@
 
     initNetworkListeners(); // Will emit Beam network call events whenever the site makes an AJAX call
 
+    async function updateCustomAttributes(payload) {
+        try {
+            const url = `/on/demandware.store/Sites-RefArch-Site/${currLocale}/Cart-UpdateBeamCustomAttributes`
```

Comment:
`request` I don't think this URL is going to work for partners out of the box. We probably need to use a relative URL, or try to generate it from the controller and include it in `pdict`

Reply:
fixed!